### PR TITLE
[14.0][MIG] partner_disable_gravatar

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: partner_disable_gravatar/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+
+In Odoo, when you enter an email address in the form view of a partner (i.e. triggered via the *onchange* on the *email* field) and this partner doesn't have any image yet in Odoo, Odoo will automatically send an HTTPS query to `www.gravatar.com <https://www.gravatar.com/>`_ with an MD5 hash of the email of the partner. If it receives an answer from gravatar.com in the form of a picture within the 5 seconds timeout, it will set this picture on the *image* field of the partner in Odoo.
+
+Some people may consider it as a cool feature to easily get picture on partners with no effort.
+
+But other people may consider this as an annoying feature that adds unnecessary network trafic or, worse, as a leak of information. With this feature, gravatar.com is notified of all the email addresses added in the Odoo database, so it may be considered as a leak of information to a third party company (gravatar.com is operated by `Automattic Inc. <https://automattic.com/contact/>`_, an American company). The problem is that there is no way to disable this feature via a configuration parameter of Odoo. This module brings a solution to this problem: once installed, it disables the feature.
```
Comment (simahawk) [inline partner_disable_gravatar/readme/DESCRIPTION.rst:9]: any proposal made on odoo/odoo to make it configurable?

**Review by skeller1 — APPROVED** [review]

**Review by remi-filament — APPROVED** [review]

**Comment by alexis-via** [comment]
Ready to merge

**Comment by NL66278** [comment]
As there really is nothing to translate in this module, can we get rid of the whole i18n directory preferably, or of all the po files at least?

**Review by NL66278 — APPROVED** [review]
:+1: Great to have this one now also in 14.0. Bizarre that standard Odoo still has no way to disable this "feature". Which actually I think should be disabled by default.

**Comment by alexis-via** [comment]
Haha, it seems you've got the super-powers @rousseldenis  !!!

**Pull request merged**